operator/src/main/resources/application.properties

```diff
@@ -1,5 +1,9 @@
 console.selector=
 
+console.deployment.default-image-tag=${quarkus.application.version}
+console.deployment.default-api-image=quay.io/streamshub/console-api:${console.deployment.default-image-tag}
+console.deployment.default-ui-image=quay.io/streamshub/console-ui:${console.deployment.default-image-tag}
```

Comment:
These properties impact the images that are applied to the console's `Deployment` resource when none are given via the `Console` CR's `spec.images.api` or `spec.images.ui` properties. 

Any thoughts on the naming in particular? I don't expect these would typically be specified very often aside from perhaps pinning a specific SHA in the operator's ClusterServiceVersion.

cc: @k-wall